queries/rust/highlights.scm

```diff
@@ -25,7 +25,10 @@
 
 (self) @variable.builtin
 
-(lifetime   ["'" (identifier)] @label)
+; This defines a new group: `@label.lifetime-only-in-rust`, which can be used to
+; highlight lifetimes separately from other labels in Rust. If not overridden,
+; it defaults to its parent: `@label`.
+(lifetime   ["'" (identifier)] @label.lifetime-only-in-rust)
```

Comment:
You can just use `@label.lifetime`. This can be mapped to highlight group either via `@label.lifetime.rust` (automatically generated) or `@label.lifetime` or just `@label`

Reply:
I'll change that then (it was suggested by someone in the Zulip server I think, since it's not a concept in other languages).